bpfd-api/src/util.rs

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: (MIT OR Apache-2.0)
+// Copyright Authors of bpfd
+
+#[rustfmt::skip]
+pub mod directories {
+    // The following directories are used by bpfd. They should be created by bpfd service
+    // via the bpfd.service settings. They will be manually created below in case bpfd
+    // is not being run as a service.
+    //
+    // ConfigurationDirectory: /etc/bpfd/
+    pub const CFGDIR:                 &str = "/etc/bpfd";
+    pub const CFGDIR_CA_CERTS:        &str = "/etc/bpfd/certs/ca";
+    pub const CFGDIR_STATIC_PROGRAMS: &str = "/etc/bpfd/programs.d";
+    pub const CFGPATH_BPFD_CONFIG:    &str = "/etc/bpfd/bpfd.toml";
+    //
+    // RuntimeDirectory: /run/bpfd/
+    pub const RTDIR:            &str = "/run/bpfd";
+    pub const RTDIR_BYTECODE:   &str = "/run/bpfd/bytecode";
+    pub const RTDIR_DISPATCHER: &str = "/run/bpfd/dispatchers";
+    pub const RTDIR_FS:         &str = "/run/bpfd/fs";
+    pub const RTDIR_FS_MAPS:    &str = "/run/bpfd/fs/maps";
+    pub const RTDIR_PROGRAMS:   &str = "/run/bpfd/programs";
+    //
+    // CacheDirectory: /var/cache/
```

Comment:
Were the `CacheDirectory` and `LogsDirectory` Vars left here by accident?

Reply:
One comment wasn't enough?

Reply:
GH glitch lol